src/geophires_x/Outputs.py

```diff
@@ -1641,7 +1641,11 @@ def PrintOutputs(self, model: Model):
                     f.write(f'      CHP: Percent cost allocation for electrical plant: {model.economics.CAPEX_heat_electricity_plant_ratio.value*100.0:10.2f} %\n')
 
                 if model.surfaceplant.enduse_option.value in [EndUseOptions.ELECTRICITY]:
-                    f.write(f'      Estimated Jobs Created:                                 {model.economics.jobs_created.value}\n')
+                    f.write(f'      Estimates from https://geothermal.org/resources/geothermal-basics:' + NL)
```

Comment:
Revert to `Estimated Jobs Created`
1. This would be a backwards-incompatible field name change
2. Although I like the idea of citing on a per-field basis, we shouldn't include citations in field names as a general principle since this would make output very difficult to read. Rather, we should include citations in documentation and/or a documentation metadata field - filed tracking issue https://github.com/NREL/GEOPHIRES-X/issues/248?title=Citation+metadata+in+parameter+descriptions
3. This specific citation is already present in the parameter description https://nrel.github.io/GEOPHIRES-X/parameters.html#id11 -> ctrl+F "Estimated Jobs Created per MW of Electricity Produced"

Reply:
Taking a second look - I see this is actually an additional line, not a replacement of estimated jobs created. Will follow up with additional thoughts on this, but making note that part of my comment is partially inaccurate

Reply:
I think that having the URL in the code is useful, but it is invisible to users who just use the UI. I think we need to show it in the UI in a backward compatible way. I will await your further comments before I change anything.